packages/less/Gruntfile.js

```diff
@@ -224,11 +220,14 @@ module.exports = function(grunt) {
             test: {
                 command: [
                     // https://github.com/TypeStrong/ts-node/issues/693#issuecomment-848907036
-                    crossEnv + " TS_NODE_SCOPE=true",
-                    tsNodeRuntime + " test/test-es6.ts",
+                    "cross-env" + " TS_NODE_SCOPE=true",
+                    "ts-node" + " test/test-es6.ts",
                     "node test/index.js"
                 ].join(' && ')
             },
+            copybrowsertestdata: {
+                command: 'npm run copy:browser-test-data'
```

Comment:
No.5 same with No.4, when we launch a nodejs server to handle browser tests request, the headless browser can't access these files, we should copy test-data files to less folder. see https://github.com/less/less.js/actions/runs/5775556003/job/15653697231#step:7:437